survey/views/survey_detail.py

```diff
@@ -63,7 +67,7 @@ def handle_invalid_form(context, form, request, survey):
         if survey.template is not None and len(survey.template) > 4:
             template_name = survey.template
         else:
-            if survey.display_by_question:
+            if survey.display_method > Survey.ALL_IN_ONE_PAGE:
```

Comment:
Actually, this is used in 4 places, this could be a function in the `Survey` class ? Maybe `not survey.is_all_in_one_page()`

Reply:
Done